sources/packages/backend/apps/api/src/services/application-exception/application-exception.service.ts

```diff
@@ -155,7 +164,21 @@ export class ApplicationExceptionService extends RecordDataModelService<Applicat
       exceptionToUpdate.assessedDate = now;
       exceptionToUpdate.modifier = auditUser;
       exceptionToUpdate.updatedAt = now;
-      return applicationExceptionRepo.save(exceptionToUpdate);
+
+      const exception = await applicationExceptionRepo.save(exceptionToUpdate);
+
+      // Create a student notification when ministry completes an exception.
+      await this.notificationActionsService.saveExceptionCompleteNotification(
+        {
+          givenNames: exceptionToUpdate.application.student.user.firstName,
```

Comment:
just a suggestion, as we did in other parts, may u can assign a user to a const and use it. 
```
const user = exception.application.student.user;

lastName: user.lastName,
 toAddress: user.email,
 userId: user.id,
```